duckduckgo/iOS
# Remove Sync Rollout banner

Task/Issue URL: https://app.asana.com/0/1177771139624306/1207276915641631/f
Tech Design URL:
CC:

**Description**:
Removes the Sync rollout banner as now Sync has been rolled out on all platforms. 



**Steps to test this PR**:
1. Make sure CI is green.
2. Verify the banner is not visible anymore in Sync Settings.





---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
